app/tests/unit/routes/v1/email.spec.js

```diff
@@ -35,6 +26,10 @@ describe(`POST ${basePath}`, () => {
   });
 
   it('should push a message and yield an Ethereal url', async () => {
+    ChesService.prototype.sendEmail.mockImplementation(() => {
```

Comment:
Since the test is on the route, you can't use this mock implementation.  You will not be testing if the route is indicating whether or not to invoke ethereal. The test should be to test if devMode=true will fire ethereal, but this always returns an ethereal url from the chesSvc.  Get this test to actually test devMode=true, and another one that tests the negative case.

Reply:
We can actually satisfy checking whether the sendEmail function was invoked to run with ethereal as true or false by employing a jest spy. Assigning the spy to the mock, we can check what arguments were passed into the mocked function.
`expect(spy).toHaveBeenCalledWith(undefined, body, true);`